action.yml

```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 name: Set up gcloud Cloud SDK environment
-author: Google LLC
+author: GoogleCloudPlatform
```

Comment:
Changes to this file need to be reverted (this is for the deprecate code)

Reply:
reverted.